auth0/src/main/java/com/auth0/android/result/Credentials.java

```diff
@@ -103,6 +103,13 @@ public String getRefreshToken() {
         return refreshToken;
     }
 
+    /**
+     * Getter for the token lifetime in seconds.
+     * Once expired, the token can no longer be used to access an API and a new token needs to be obtained.
+     *
+     * @return the token lifetime in seconds.
```

Comment:
"since the time it was returned".
but is this any useful? I know the spec uses 'expires in' but maybe it's better if it's calculated? you save the calculated expiration time when the token is created, and then return "exp minus now" ? 
or directly add a method to return the approximate expiration time and delete this one?